<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="layoutX">
       <item>
        <widget class="QLabel" name="lab_Xmin">
         <property name="text">
          <string>XMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditXmin"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Xmax">
         <property name="text">
          <string>XMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditXmax"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutY">
       <item>
        <widget class="QLabel" name="labelYmin">
         <property name="text">
          <string>Ymin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditYmin"/>
       </item>
       <item>
        <widget class="QLabel" name="labelYmax">
         <property name="text">
          <string>Ymax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditYmax"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutZ">
       <item>
        <widget class="QLabel" name="labelZmin">
         <property name="text">
          <string>Zmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZmin"/>
       </item>
       <item>
        <widget class="QLabel" name="labelZmax">
         <property name="text">
          <string>Zmax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZmax"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
